bot/cogs/reddit.py

```diff
@@ -55,6 +57,8 @@ def __init__(self, bot: Bot):
                 content = await response.json()
                 posts = content["data"]["children"]
                 return posts[:amount]
+            async with channel.typing():
```

Comment:
If this is done instead in `send_top_posts` and wraps the `fetch_posts` call, the signature of `fetch_posts` wont need to be modified.

Reply:
Alrighty. I'm real new to GitHub and collaboration in general - is it preferable for me to revert the last commit and make the change to `send_top_posts`, or for me to leave git alone and just use a new commit to put the code back how it was and implement the change to `send_top_posts`?

Reply:
Do whatever is easier for you. I generally avoid reverts in these situations because they're just an extra step.

Reply:
@bendiller You should probably just make a new commit that addresses the review comments, rather than mucking around with reverts or amends.

Reply:
Thanks for the feedback. I implemented the the `async with channel.typing()...` behavior in `send_top_posts` instead as you suggested. I left it out of the `fetch_posts` call in `poll_new_posts`, because in that case there's no user to inform of the delay. I can change that, if you prefer.

Reply:
Nope, no need to add it to the poll functions. This looks good.